README.md

```diff
@@ -95,211 +96,264 @@ Specifying those two environment variables will override authentication passed i
+| elasticsearch_health_report_ilm_policies                             | gauge   | 1           | The number of ILM Policies                                                                          |
+| elasticsearch_health_report_ilm_stagnating_indices                   | gauge   | 1           | The number of stagnating indices                                                                    |
+| elasticsearch_health_report_ilm_status                               | gauge   | 2           | ILM status                                                                                          |
+| elasticsearch_health_report_initializing_primaries                   | gauge   | 1           | The number of initializing primary shards                                                           |
+| elasticsearch_health_report_initializing_replicas                    | gauge   | 1           | The number of initializing replica shards                                                           |
+| elasticsearch_health_report_master_is_stable_status                  | gauge   | 2           | Master is stable status                                                                             |
+| elasticsearch_health_report_max_shards_in_cluster_data               | gauge   | 1           | The number of maximum shards in a cluster                                                           |
+| elasticsearch_health_report_max_shards_in_cluster_frozen             | gauge   | 1           | The number of maximum frozen shards in a cluster                                                    |
+| elasticsearch_health_report_repository_integrity_status              | gauge   | 2           | Repository integrity status                                                                         |
+| elasticsearch_health_report_restarting_primaries                     | gauge   | 1           | The number of restarting primary shards                                                             |
+| elasticsearch_health_report_restarting_replicas                      | gauge   | 1           | The number of restarting replica shards                                                             |
+| elasticsearch_health_report_shards_availabilty_status                | gauge   | 2           | Shards availabilty status                                                                           |
+| elasticsearch_health_report_shards_capacity_status                   | gauge   | 2           | Shards capacity status                                                                              |
+| elasticsearch_health_report_slm_policies                             | gauge   | 1           | The number of SLM policies                                                                          |
+| elasticsearch_health_report_slm_status                               | gauge   | 2           | SLM status                                                                                          |
```

Comment:
Can you please pull the whitespace changes out? They could be in a different PR but they're unrelated here and make it harder to see the actual changes.

Reply:
Done